code/modules/projectiles/projectile/beams.dm

```diff
@@ -26,8 +26,8 @@
 	eyeblur = 2
 
 /obj/item/projectile/beam/smalllaser
-	damage = 25
-	armor_penetration = 5
+	damage = 30
```

Comment:
```suggestion
	damage = 25
```

Reply:
Tested it again, 30 was too strong and could end lungs in 2-3 shots. 25 takes 5 shots average to damage lungs badly